.gitignore

```diff
@@ -0,0 +1,139 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
General comment, have we considered using https://github.com/ni/nisystemlink-clients-python as a starting point for all of this?